kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -54,4 +74,21 @@ describe('Overview', () => {
       expect(componentEmpty.find('td').text()).toEqual('No Partitions found');
     });
   });
+
+  describe('should render circular alert', () => {
+    it('should be in document', () => {
+      renderComponent();
+      const circles = screen.getAllByRole('circle');
+      expect(circles[0]).toBeInTheDocument();
+      expect(circles[1]).toBeInTheDocument();
+      expect(circles.length).toEqual(2);
+    });
+
+    it('should be the appropriate color', () => {
+      renderComponent();
+      const circles = screen.getAllByRole('circle');
+      expect(circles[0]).toHaveStyle(`fill: ${Colors.green[40]}`);
```

Comment:
As Oleg mentioned in my PR, that we should not use explicitly 'Colors', but import them as theme, I think that here we should also use theme like this:
https://github.com/provectus/kafka-ui/blob/master/kafka-ui-react-app/src/components/common/Pagination/__tests__/Pagination.spec.tsx#L83

Reply:
Oops, you're right, thanks. 😊